assets/mods/simplify/mod.js

```diff
@@ -804,7 +807,18 @@
 						this.addChildGui(this.iconGui);
 						this.nameGui.hook.pos.x += this.iconGui.hook.pos.x + this.iconGui.hook.size.x;
 					}
-			
+
+					if(this.option.version != null) {
+						this.versionText = new sc.TextGui(this.option.version, {
+							font: sc.fontsystem.tinyFont,
+							speed: ig.TextBlock.SPEED.IMMEDIATE
+						});
+						this.versionText.setAlign(ig.GUI_ALIGN.X_RIGHT, ig.GUI_ALIGN.Y_BOTTOM);
+						console.log(this.hook.size.x, this.hook.size.x / 2, this.hook.size.x - 48, this.hook.size.y)
+						this.versionText.setPos(38, 5);
```

Comment:
Can you please calculate this position from other coordinates? Like this:

```js
let lineHook = this.hook.children[1]; // this is already defined a little above
let slopeHook = this.hook.children[2];
// ...
// this code:
this.versionText.setPos(slopeHook.pos.x - this.versionText.hook.size.x - 4, slopeHook.pos.y + 2);
```

Reply:
Oh, and please remove the `console.log`

Reply:
🤦 I thought I removed that... And yeah I can see about that.